grafana-stack/influxdb/influxdb.toml

```diff
@@ -0,0 +1,124 @@
+reporting-disabled = false
+bind-address = ":8088"
+hostname = ""
+join = ""
+
+[meta]
+  dir = "/root/.influxdb/meta"
+  retention-autocreate = true
+  logging-enabled = true
+  pprof-enabled = false
+  lease-duration = "1m0s"
+
+[data]
+  dir = "/root/.influxdb/data"
+  engine = "tsm1"
+  wal-dir = "/root/.influxdb/wal"
+  wal-logging-enabled = true
+  query-log-enabled = true
+  cache-max-memory-size = 524288000
+  cache-snapshot-memory-size = 26214400
+  cache-snapshot-write-cold-duration = "1h0m0s"
+  compact-full-write-cold-duration = "24h0m0s"
+  max-points-per-block = 0
+  data-logging-enabled = true
+
+[cluster]
+  force-remote-mapping = false
+  write-timeout = "5s"
+  shard-writer-timeout = "5s"
+  max-remote-write-connections = 3
+  shard-mapper-timeout = "5s"
+  max-concurrent-queries = 0
+  query-timeout = "0"
+  max-select-point = 0
+  max-select-series = 0
+  max-select-buckets = 0
+
+[retention]
+  enabled = true
+  check-interval = "30m0s"
+
+[shard-precreation]
+  enabled = true
+  check-interval = "10m0s"
+  advance-period = "30m0s"
+
+[admin]
+  enabled = true
+  bind-address = ":8083"
+  https-enabled = false
+  https-certificate = "/etc/ssl/influxdb.pem"
+  Version = ""
+
+[monitor]
+  store-enabled = true
+  store-database = "_internal"
+  store-interval = "10s"
+
+[subscriber]
+  enabled = true
+
+[http]
+  enabled = true
+  bind-address = ":8086"
+  auth-enabled = true
+  log-enabled = true
+  write-tracing = false
+  pprof-enabled = false
+  https-enabled = false
+  https-certificate = "/etc/ssl/influxdb.pem"
+  max-row-limit = 10000
+
+[[graphite]]
```

Comment:
No idea.
You can't provide infludb with a subset of a configuration file, it needs to be complete. So you are supposed to do
`influxdb config >generated-config`. And that happened.

The only field I changed is: `[http] auth_enabled = true`